src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carRent = 40;
+  let fullCost = days * carRent;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+
+  if (days >= LONG_TERM) {
+    fullCost -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
```

Comment:
You can remove the `else` keyword since the `if` block contains a `return` statement. This simplifies the control flow and makes the code easier to read.